### ISSUE TITLE
JAWS: The color picker in Table properties is labelled as "Unlabeled 0"

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#11895]
## 📝 Provide detailed reproduction steps (if any)

1.  Insert table.
2.  Click Table properties.
3.  Start JAWS, use CTRL+Insert+C to get the available list of controls

### ✔️ Expected result

The color picker is labelled properly.

### ❌ Actual result

The color picker is labelled as "Unlabeled 0"

![](https://user-images.githubusercontent.com/20988892/172591192-97790ba9-38a5-4b7b-a6c1-d1a68103cfac.png)

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by mlewand** [comment]
The color dropdown button has a following markup:

```
<button
	class="ck ck-button ck-off ck-dropdown__button ck-input-color__button"
	type="button"
	tabindex="-1"
	aria-labelledby="ck-editor__aria-label_e3335aa5a5c2924ea39541dacfea32bb9"
	aria-haspopup="true">
```

Where `aria-labelledby` leads to an empty element (so there's nothing to read).

Whereas for sighted users it has a label:

![](https://user-images.githubusercontent.com/5353898/173851073-8f79c2a0-711c-4e1c-95e4-eeea1d11d862.png)

The same label ("Color picker") should be given to the screen reader.

**Comment by mlewand** [comment]
@Mgsy can you confirm whether fix in #11963 fixes this issue?
